source/regex.tex

```diff
@@ -1240,29 +1240,29 @@
       basic_regex();
       explicit basic_regex(const charT* p, flag_type f = regex_constants::ECMAScript);
       basic_regex(const charT* p, size_t len, flag_type f = regex_constants::ECMAScript);
-      basic_regex(const basic_regex&);
-      basic_regex(basic_regex&&) noexcept;
+      basic_regex(const basic_regex& e);
+      basic_regex(basic_regex&& e) noexcept;
```

Comment:
We're fairly inconsistent with naming parameters in class synopses; see e.g. [vector.overview]. I'm ok with the rest of the renames, but why do we need a parameter name here?

Reply:
I didn't notice before the standards wasn't really consistent in the naming. Do you want me to remove this hunk of the diff?

Reply:
Yes, please. (Force-pushing is fine.) I actually believe that constructors and assignment operators should not have parameter names in synopses; there is no additional information conveyed in those parameter names and the signatures should otherwise speak for themselves.